src/Illuminate/View/Compilers/BladeCompiler.php

```diff
@@ -565,8 +565,10 @@ public function component($class, $alias = null, $prefix = '')
         }
 
         if (is_null($alias)) {
-            $alias = Str::contains($class, '\\View\\Components\\')
-                            ? collect(explode('\\', Str::after($class, '\\View\\Components\\')))->map(function ($segment) {
+            $namespace = config('view.component_namespace', 'View\\Components');
```

Comment:
You cannot call that function here.

Reply:
Does this compiler fire before the config values are processed?